PROCESSORS.md

```diff
@@ -421,8 +421,9 @@ In the list below, the names of required properties appear in bold. Any other pr
 
 | Name | Default Value | Allowable Values | Description |
 | - | - | - | - |
-|Script File|||Path to script file to execute. Only one of Script File or Script Body may be used|
+|**Reload on Script Change**|false||If true and Script File property is used, then script file will be reloaded if it has changed, otherwise the first loaded version will be used at all times.|
```

Comment:
My only concern was the backward compatibility especially for native python processors as previously even though it looked like we reloaded the script actually it was never reloaded. But on second thought I don't think it would be a problem and we still haven't released version 1.0 so I updated it in 3a8c9b43245739352f3fef8602dac1592f0eafdd

Reply:
Could you elaborate on why was it never reloaded? I thought it was reloaded whenever the last write time increased.

Reply:
The PythonScriptEngine's eval function was not called after reading the file's content.